[Node.js] Cannot find module 'tokenizers-darwin-arm64' after npm install on macOS (Apple Silicon)

I’m encountering an issue when installing and using the tokenizers npm package on macOS with Apple Silicon (ARM64).
After running npm install, the installation completes without fatal errors, but at runtime Node.js fails to load the native binary:
```
Error: Cannot find module 'tokenizers-darwin-arm64'
```
This happens immediately when importing or requiring tokenizers.

Environment:

OS: macOS (Apple Silicon, ARM64)
Node.js version: 25
npm version: 11.6.2
tokenizers version: 0.13.3
Package manager: npm
Architecture: arm64

--- a/.github/workflows/node-release.yml
+++ b/.github/workflows/node-release.yml
@@ -2,7 +2,7 @@ name: Node Release
 
 env:
   APP_NAME: tokenizers
-  MACOSX_DEPLOYMENT_TARGET: '10.13'
+  MACOSX_DEPLOYMENT_TARGET: '12.0'
 
 permissions:
   contents: read
@@ -68,7 +68,7 @@ jobs:
     name: build - ${{ matrix.settings.target }}
     runs-on: ${{ matrix.settings.host }}
     steps:
-      - uses: actions/checkout@v4
+      - uses: actions/checkout@v6
 
       - name: Install Rust
         uses: dtolnay/rust-toolchain@29eef336d9b2848a0b548edc03f92a220660cdb8 # stable
@@ -78,7 +78,7 @@ jobs:
           targets: ${{ matrix.settings.target }}
 
       - name: Setup node
-        uses: actions/setup-node@v4
+        uses: actions/setup-node@v6
         if: ${{ !matrix.settings.docker }}
         with:
           node-version: lts/*
@@ -92,7 +92,7 @@ jobs:
 
       - name: Setup zig
         if: ${{ !matrix.settings.docker && contains(matrix.settings.build, '--zig') }}
-        uses: goto-bus-stop/setup-zig@abea47f85e598557f500fa1fd2ab7464fcb39406 # v2.2.1
+        uses: https://codeberg.org/mlugg/setup-zig@v2
 
       - name: Build native (host)
         if: ${{ !matrix.settings.docker }}
@@ -101,7 +101,7 @@ jobs:
 
       - name: Build native (docker)
         if: ${{ matrix.settings.docker }}
-        uses: addnab/docker-run-action@4f65fabd2431ebc8d299f8e5a018d79a769ae185 # v3
+        uses: addnab/docker-run-action@v3
         with:
           image: ${{ matrix.settings.docker }}
           options: >-
@@ -114,7 +114,7 @@ jobs:
             ${{ matrix.settings.build }}
 
       - name: Upload artifact
-        uses: actions/upload-artifact@v4
+        uses: actions/upload-artifact@v7
         with:
           name: bindings-${{ matrix.settings.target }}
           path: bindings/node/*.node
@@ -125,9 +125,9 @@ jobs:
     needs: build
     runs-on: macos-latest
     steps:
-      - uses: actions/checkout@v4
+      - uses: actions/checkout@v6
       - name: Setup node
-        uses: actions/setup-node@v4
+        uses: actions/setup-node@v6
         with:
           node-version: lts/*
           cache: yarn
@@ -136,12 +136,12 @@ jobs:
         working-directory: ./bindings/node
         run: yarn install --immutable
       - name: Download macOS x86_64 artifact
-        uses: actions/download-artifact@v4
+        uses: actions/download-artifact@v8
         with:
           name: bindings-x86_64-apple-darwin
           path: ./bindings/node/artifacts
       - name: Download macOS aarch64 artifact
-        uses: actions/download-artifact@v4
+        uses: actions/download-artifact@v8
         with:
           name: bindings-aarch64-apple-darwin
           path: ./bindings/node/artifacts
@@ -149,7 +149,7 @@ jobs:
         working-directory: ./bindings/node
         run: npx napi universal --name ${{ env.APP_NAME }}
       - name: Upload universal artifact
-        uses: actions/upload-artifact@v4
+        uses: actions/upload-artifact@v7
         with:
           name: bindings-universal-apple-darwin
           path: bindings/node/${{ env.APP_NAME }}.*.node
@@ -165,9 +165,9 @@ jobs:
       - build
       - universal-macOS
     steps:
-      - uses: actions/checkout@v4
+      - uses: actions/checkout@v6
       - name: Setup node
-        uses: actions/setup-node@v4
+        uses: actions/setup-node@v6
         with:
           node-version: lts/*
           cache: yarn
@@ -176,7 +176,7 @@ jobs:
         working-directory: ./bindings/node
         run: yarn install --immutable
       - name: Download all artifacts
-        uses: actions/download-artifact@v4
+        uses: actions/download-artifact@v8
         with:
           path: ./bindings/node/artifacts
       - name: Move artifacts

